### ISSUE TITLE
Hompage Styles - Create and style bottom section in homepage

### ISSUE THREAD

**Issue by fridaland** [fridaland/InTheFight#26]
### Create and style bottom section in homepage

Add bottom section made up of **2 elements**.  This section should appear at the bottom of the homepage.
<img width="685" alt="Screen Shot 2020-10-23 at 11 18 19 AM" src="https://user-images.githubusercontent.com/40510669/97028740-280dda80-1522-11eb-96ce-3fbb26cfd515.png">

The image for the bottom section can be found under `src/images/home-images`.
Use:
- `footer-image.png`

##### First Container 

Title and description for first container:
```
Header
Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.

<button>Action</button>
```

Spec styles for first container:

```
Container: 
height: 436px;
background: #2C358F;

Header:
font-family: Gelo;
font-size: 32px;
text-align: center;
color: #FAF5EB;


Description:
font-family: Gelo;
font-size: 18px;
text-align: center;
color: #FAF5EB;

Button:
background: #FAF5EB;
color: #2C358F;
```

##### Second Container 

Description for second container:
```
Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.

- lorem Ipsum Author
```

Spec styles for second container:
```
Container:

height: 436px;

Background image : `footer-image.png`
(You can likely use gatsby-background-image package)

-  Above quote description should sit on top of background image

Text styles:

font-family: URW DIN;
font-weight: 300;
font-size: 20px;
color: #FAF5EB;

```
See attachments below:
<img width="253" alt="Screen Shot 2020-10-23 at 11 05 35 AM" src="https://user-images.githubusercontent.com/40510669/97027274-175c6500-1520-11eb-89cb-926338c32c5e.png">

**Comment by wandersonpsantos** [comment]
can i make this issue?

**Comment by fridaland** [comment]
@wandersonpsantos It's yours!

**Comment by fridaland** [comment]
Resolved with #32
